scripts/opensearch-onetime-setup.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+
+# This script performs one-time setup for the OpenSearch tarball distribution.
+# It installs a demo security config and sets up the performance analyzer
+
+OPENSEARCH_HOME=`dirname $(realpath $0)`; cd $OPENSEARCH_HOME
+
+##Security Plugin
+SECURITY_PLUGIN="opensearch-security"
+bash $OPENSEARCH_HOME/plugins/$SECURITY_PLUGIN/tools/install_demo_configuration.sh -y -i -s
+
+##Perf Plugin
+PA_PLUGIN="opensearch-performance-analyzer"
```

Comment:
You don't need this either.

Reply:
Need more debug and possible changes on docker then.

Reply:
Not sure why GitHub included line 34 in the context here...the comment was specifically referring to the PA_PLUGIN variable. It's only used once, on line 38, and not exported, so you could just change line 38 to say "opensearch-performance-analyzer" rather than $PA_PLUGIN.

You definitely do need line 34.